rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -417,6 +417,9 @@ http {
         {{ if (gt $cfg.UpstreamKeepaliveConnections 0) }}
         keepalive {{ $cfg.UpstreamKeepaliveConnections }};
         {{ end }}
+
+        keepalive_timeout  {{ $cfg.UpstreamKeepAliveTimeOut }}s;
+        keepalive_requests {{ $cfg.UpstreamKeepAliveRequests }};
```

Comment:
should this not be inside the if condition above?

Reply:
It does make the configuration cleaner!